src/renderer/component/walletBalance/view.jsx

```diff
@@ -10,10 +10,11 @@ const WalletBalance = (props: Props) => {
   const { balance } = props;
   return (
     <section className="card card--section card--wallet-balance">
-      <div>
-        <div className="card__title">{__('Balance')}</div>
-        <span className="card__subtitle">{__('You currently have')}</span>
-      </div>
+      <header className="card__header">
```

Comment:
I thought we weren't worrying about making things responsive.

Reply:
Why are we showing all these decimal points? We don't in the header bar. We don't in the hover text either.

```sass
display: block;
max-width: 400px;
overflow: hidden;
padding-right: 7rem;
text-overflow: ellipsis;
white-space: nowrap;
```

This does not look good on the `<span/>` surrounding the amount. You have ellipsis and extra space to add "LBC" in an `::after` pseudo-element.

Another option is to have the numbers fade (to replace the ellipsis).

Let me know which direction you want me to go in.

Reply:
We aren't worrying about mobile design, but the minimum app width still needs to look good.

We should show the full amount here, it's the only place you can see it. I think it's fine to show an estimate in the header because it goes to this page when you click on it, and shows the full amount in the hover text. It's a bug that we show an estimate when hovering over the large balance (I guess we don't even really need that though). 

What about just dropping the `font-size` down to `4rem`?

Reply:
I'll make it `3.5rem` so the Rewards box has more breathing room at `950px`.